terraform/main.tf

```diff
@@ -0,0 +1,24 @@
+terraform {
+  required_providers {
+    opennebula = {
+      source  = "OpenNebula/opennebula"
+      version = "~> 1.4"
+    }
+  }
+}
+
+provider "opennebula" {
+  endpoint      = var.endpoint
+  flow_endpoint = var.flow_endpoint
+  username      = var.username
+  password      = var.password
+  insecure      = true
```

Comment:
Why insecure = true?
I have tested on CL environment and it`s works witout this parameter